### ISSUE TITLE
iocage fetch not parsing correctly

### ISSUE THREAD

**Issue by ChaosSteffen** [iocage/iocage#299]
After having this error when using 0.9.8.1 from ports, I've build from master and had the same issue.

Looks like the parsing of the ftp directory does have some issues (see listing)

Installing with `-r` still works fine.

```
beast# iocage fetch -NE
[0] 1
[0] 1
[2] .
[0] 1
[4] -
[5] R
[6] E
[7] L
[6] E
[9] A
[10] S
[6] E

Type the number of the desired RELEASE
Press [Enter] to fetch the default selection: (11.0-RELEASE)
Type EXIT to quit: 1
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/ioc_fetch.py", line 461, in fetch_ftp_release
    ftp.cwd(self.release)
  File "/usr/local/lib/python3.6/ftplib.py", line 629, in cwd
    return self.voidcmd(cmd)
  File "/usr/local/lib/python3.6/ftplib.py", line 276, in voidcmd
    return self.voidresp()
  File "/usr/local/lib/python3.6/ftplib.py", line 249, in voidresp
    resp = self.getresp()
  File "/usr/local/lib/python3.6/ftplib.py", line 244, in getresp
    raise error_perm(resp)
ftplib.error_perm: 550 Failed to change directory.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/iocage", line 10, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/iocage/cli/fetch.py", line 95, in cli
    ioc.IOCage().fetch(**kwargs)
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/iocage.py", line 754, in fetch
    ioc_fetch.IOCFetch(release, **kwargs).fetch_release()
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/ioc_fetch.py", line 264, in fetch_release
    self.fetch_ftp_release(eol, _list=_list)
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/ioc_fetch.py", line 463, in fetch_ftp_release
    raise RuntimeError(f"{self.release} was not found!")
RuntimeError: 1 was not found!
```

**Comment by ChaosSteffen** [comment]
BTW: Was trying to fix this on my own but then realised that #267 is going to change a lot of code and postponed it. Once you merge I can have a look again.

**Comment by skarekrow** [comment]
I'm going to try fixing this before 0.9.9 and a couple other things if memory serves. Expect 0.9.9 around Friday if all goes well.
